### PR TITLE
Fix #11790: Reset button added for arrange property

### LINKED ISSUE #11790: [Inspector] Reset option is missing in Properties>Appearance>Arrange section
**Describe the bug**
See compare with MU3

https://user-images.githubusercontent.com/90187801/170735311-efe18eda-7adf-4ae0-aca0-e9b561e1ae3f.mp4

Untitled

**Platform information**
macOS, Windows, Linux

**Additional context**
Doesn't occur in **MU3** (in MU3 it's called Reset to style default)

MU4 Backward/Forwards buttons in Arrange section are equivalent of Stacking order (Z) feature in MU3

### PR DESCRIPTION
**Pull request by mathesoncalum**
Resolves: #11790

A new reset button has been added for the Properties->Appearance->Arrange panel

### CODE REVIEW

File: src/inspector/view/qml/MuseScore/Inspector/general/appearance/internal/ArrangeSection.qml [hunk 1]
```diff
@@ -46,10 +51,27 @@ Column {
...
+            }
+
+            PropertyResetButton {
+                anchors.top: parent.top
+                anchors.right: parent.right
+
+                navigation.name: root.navigationName + "Reset"
+                navigation.panel: root.navigationPanel
+                navigation.row: root.buttonNavigationRow
+                navigation.accessible.name: qsTrc("inspector", "Reset order to default value")
```
Comment (cbjeukendrup) [inline src/inspector/view/qml/MuseScore/Inspector/general/appearance/internal/ArrangeSection.qml:71]: Maybe "Reset _arrangement_ to default" might be clearer, but I'm not sure

File: src/inspector/view/qml/MuseScore/Inspector/common/InspectorPropertyView.qml [hunk 2]
```diff
@@ -120,24 +120,16 @@ Column {
...
-                    width: 20
-                    height: width
-
+                PropertyResetButton {
                     navigation.name: root.navigationName + "Reset"
                     navigation.panel: root.navigationPanel
                     navigation.row: root.buttonNavigationRow
                     navigation.accessible.name: root.titleText ? qsTrc("inspector", "Reset \"%1\" to default value").arg(root.titleText)
-                                                               : qsTrc("inspector", "Reset property to default value")
+                                                                              : qsTrc("inspector", "Reset property to default value")
```
[the remark this answers is not in the document]
Comment (mathesoncalum) [reply src/inspector/view/qml/MuseScore/Inspector/common/InspectorPropertyView.qml:128]: @cbjeukendrup Thanks for these! I'll get to it ☺️

File: src/inspector/view/qml/MuseScore/Inspector/general/appearance/internal/ArrangeSection.qml (hunk 1 above)
Comment (mathesoncalum) [reply src/inspector/view/qml/MuseScore/Inspector/general/appearance/internal/ArrangeSection.qml:71]: @cbjeukendrup I was wondering about this actually - I think in MS3 it was called "stacking order" which might be an option?

Comment (cbjeukendrup) [reply src/inspector/view/qml/MuseScore/Inspector/general/appearance/internal/ArrangeSection.qml:71]: Ah, yes, that might be the best candidate actually!

File: src/inspector/view/qml/MuseScore/Inspector/general/appearance/internal/ArrangeSection.qml [hunk 3]
```diff
@@ -46,10 +53,29 @@ Column {
...
+
+            StyledTextLabel {
+                width: parent.width
+                text: qsTrc("inspector", "Arrange")
+                horizontalAlignment: Text.AlignLeft
+            }
+
+            PropertyResetButton {
+                anchors.top: parent.top
+                anchors.right: parent.right
```
Comment (RomanPudashkin) [inline src/inspector/view/qml/MuseScore/Inspector/general/appearance/internal/ArrangeSection.qml:70]: Now you can remove these anchors

**Comment by DmitryArefiev** [comment]
Tested #11790 on Win10, Mac13, LinuxUbuntu 22.04 - **FIXED**

**Pull request merged**